kubernetes/kubernetes.go

```diff
@@ -91,7 +91,25 @@ const direct = "direct"
 const unreachable = "unreachable"
 
 //nolint llll
-const kbURL string = "https://support.cloudability.com/hc/en-us/articles/360008368193-Kubernetes-Metrics-Agent-Error-Messages"
+const kbErrorURL string = "https://help.apptio.com/en-us/cloudability/product/k8s-metrics-agent.htm"
+const kbProvisionURL string = "https://help.apptio.com/en-us/cloudability/product/k8s-cluster-provisioning.htm"
+const apiEndpoint string = "https://metrics-collector.cloudability.com"
+
+const forbiddenError string = "Error retrieving upload URI: 403"
+const uploadURIError string = "Error retrieving upload URI"
+const transportError string = "Network transport issues are potentially blocking the agent from " +
+	"contacting the metrics collection API.\n" +
+	"Please confirm that the metrics-agent is able to establish a connection to: %s"
+const apiKeyError string = "Current Cloudability API Key needs to be re-enabled before re-provisioning " +
+	"the metrics-agent as detailed here: %s.\n" +
+	"Please contact support to re-activate the API keys.\n" +
+	"Note 1: Be sure to use the exact same cluster name as what is currently in use.\n" +
+	"Note 2: If the cluster is managed by GKE - there are special instructions for provisioning"
+const rbacError string = "RBAC role in the Cloudability namespace may need to be updated.\n" +
+	"Re-provision the metrics-agent for this cluster as detailed here: %s\n" +
+	"Note: Be sure to use the exact same cluster name as what is currently in use.\n" +
+	"***NOTE*** Please note that if the cluster is managed by GKE, " +
+	"there are special instructions for provisioning.\n"
```

Comment:
⭐ 
I think these error messages would be slightly easier to read if we used multiline strings instead of concatenation. What do you think?

Also, can we be consistent with how we're adding notes in these errors? There are 3 different formats currently - `Note 1/2`, `Note`, and `***NOTE***`...Hmm looking closer, the `apiKeyError` and `rbacError` both have the same notes about GKE and same cluster naming - what do you think about cleaning up both of these error messages and use the same messaging for both error's "notes"? I'm thinking this "notes" string (kinda like an error "footer" to be suffixed on each error) could look something like: 
```
Be sure to use the exact same cluster name as what is currently in use. 
***Please note that if the cluster is managed by GKE, there are special instructions for provisioning.***
```

Reply:
I think the `***NOTE***` is to indicate that this is a really important thing to pay attention to.  I can make the other one consistent though.

Reply:
Also, a lot of the concatenation is because they're one long sentence, so the linter gets cranky because of the length.  I'll see if I can make them multiline (and get rid of the newlines), but it doesn't make sense to break full up sentences into multiple lines.